spec/controllers/recordings_controller_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::RecordingsController, type: :controller do
+  let(:user) { create(:user) }
+
+  before do
+    request.headers['ACCEPT'] = 'application/json'
+    session[:user_id] = user.id
+  end
+
+  # TODO: Hadi - Make test work with current_user
+  describe 'index' do
+    it 'ids of recordings in response are matching recording ids that belong to current_user rooms' do
```

Comment:
The name of this doesnt doesnt really make sense. "it ids of recordings..."

Should instead be something like `it 'returns recordings ids that belong to current_user'`